x-pack/docs/en/security/authentication/configuring-pki-realm.asciidoc

```diff
@@ -2,26 +2,37 @@
 [[configuring-pki-realm]]
 === Configuring a PKI realm
 
-You can configure {es} to use Public Key Infrastructure (PKI) certificates
-to authenticate users. This requires clients to present X.509 certificates.
+You can configure {es} to use Public Key Infrastructure (PKI) certificates to
+authenticate users. This requires clients connecting directly to {es} to
+present X.509 certificates. The certificates must first be accepted for
+authentication on the SSL/TLS layer on {es}, before they are optionally further
+validated by a PKI realm. Users might also connect to {kib}, where they present
+their certificate, but this requires a little
+<<pki-realm-for-proxied-clients,extra configuration>>. On {es}, this
+configuration enables {kib} to act as a proxy for SSL/TLS authentication and
+submit the client certificates to {es} to be further validated by a PKI realm.
 
-NOTE: You cannot use PKI certificates to authenticate users in {kib}.
+For more general information, see {stack-ov}/pki-realm.html[PKI User Authentication].
+
+[float]
+[role="xpack"]
+[[pki-realm-for-direct-clients]]
+==== PKI authentication for clients connecting directly to {es}
 
 To use PKI in {es}, you configure a PKI realm, enable client authentication on
-the desired network layers (transport or http), and map the Distinguished Names
-(DNs) from the user certificates to roles in the 
-<<security-api-role-mapping,role-mapping API>> or role-mapping file.
+the desired network layers (transport or http), and map the Distinguished Name
+(DN) from the Subject field in the user certificate to roles, using the
+<<security-api-role-mapping,role-mapping API>> or the role-mapping file.
 
 You can also use a combination of PKI and username/password authentication. For
 example, you can enable SSL/TLS on the transport layer and define a PKI realm to
 require transport clients to authenticate with X.509 certificates, while still
-authenticating HTTP traffic using username and password credentials. You can 
-also set `xpack.security.transport.ssl.client_authentication` to `optional` to 
+authenticating HTTP traffic using username and password credentials. You can
+also set `xpack.security.transport.ssl.client_authentication` to `optional` to
 allow clients without certificates to authenticate with other credentials.
 
-IMPORTANT:  You must enable SSL/TLS and enable client authentication to use PKI.
-
-For more information, see {stack-ov}/pki-realm.html[PKI User Authentication].
+IMPORTANT: If you only allow PKI authentication for clients connecting directly
```

Comment:
It is unclear to me from the placement of the "only" what is prohibited here.   Is one of these right?:

> If clients must be authenticated by PKI in order to connect directly to {es} ...

> If PKI authentication is required only by clients connecting directly to {es}...

Reply:
Neither 🤦‍♂ 
I've rephrased it as:
```
IMPORTANT: You must enable SSL/TLS and enable client authentication, if you
only allow PKI authentication for clients connecting directly to {es}, but not
through {kib}.
```